src/include/catalog/storage_pending_deletes.h

```diff
@@ -0,0 +1,41 @@
+/*-------------------------------------------------------------------------
+ *
+ * storage_pending_deletes.h
+ *	  prototypes for functions in backend/catalog/storage_pending_deletes.c
+ *
+ * Copyright (c) 2025 Greengage Community
+ *
+ * IDENTIFICATION
```

Comment:
Do we really need this line? Headers from PostgreSQL doesn't contain it. See src/include/catalog/index.h, src/include/catalog/pg_aggregate.h, src/include/catalog/pg_trigger.h